src/types/modals/addDebtModal.ts

```diff
@@ -0,0 +1,3 @@
+export type AddDebtModalProps = {
+  returnOrTakeLoan?: boolean;
```

Comment:
It will be better to rename this prop or to make like this:
```suggestion
  action?: 'return' | 'take'; 
```